### ISSUE TITLE
Expand abreviations failed in some parts of html files

### ISSUE THREAD

**Issue by mistergraphx** [Defman21/Xemmet#25]
### Short summary

Hi, first thanks for your work on this addon.

I use a template engine from a CMS (SPIP), the syntax of loops is near xml tags, on certains parts the expand abreviation failed, and in others part of file  (outside of a loop, it seems) il works.

i know that i'm in a minority of users , but if it can improve the stability of your code i can send you the file to test and reproduce the use case.

Thanks.

### Information

_Xemmet version:_ 

0.8.0-alpha-2

_OS:_ 

Mac OsX 10.9

_Komodo version:_ 

Komodo Edit, version 10.2.0-alpha1

**Comment by Defman21** [comment]
Yes please, send me an example and some steps to reproduce it :)

**Comment by mistergraphx** [comment]
Thanks for the interest, and quick answer ;-) When you use Emmet since a long time you can't realize how you did without before lol

```html
<!-- Here emmet work div>ul>li -->

<BOUCLE_content(RUBRIQUES){id_rubrique}>
<!-- Here emmet work div>ul>li -->

<section>
	<header class="cartouche">
		[(#LOGO_RUBRIQUE|image_reduire{200,200})]
		<h1 class="#EDIT{titre}">#TITRE</h1>
		[<p class="publication">
			<time datetime="[(#DATE|date_iso)]"><:dernier_ajout:> : (#DATE|affdate_jourcourt).</time>
		</p>]
	</header>
	<!-- Here emmet work div>ul>li -->
	<div class="main">
		[<div class="#EDIT{texte} chapo">(#TEXTE)</div>]
		
		<!-- Here emmet work div>ul>li -->

		[(#REM)<!-- Menu des Sous Rubriques
		
		filtrage des travaux
		http://isotope.metafizzy.co/layout-modes/masonry.html
		http://masonry.desandro.com/
		
		
		https://jsfiddle.net/Xeoncross/kQpAf/
		
		-->]
		
		<!-- Here emmet don't work div>ul>li : return :
		div>ul>li	(just a tab)-->
				
		<B_listSousRubs>
		<nav>
			<ul>
			<BOUCLE_listSousRubs(RUBRIQUES){id_parent}>
				<li>
					<a href="[(#SELF)]">#TITRE</a>
					<!-- Here emmet don't work div>ul>li : return
					div>ul>li	(just a tab)-->
					
				</li>
			</BOUCLE_listSousRubs>
			</ul>
		</nav>
		</B_listSousRubs>
		<!-- Here emmet don't work div>ul>li : return :
		div>ul<div></div>
		-->
		
		[(#REM)<!-- Affichage de tout les docs -->]
		<BOUCLE_filtreSecteur(RUBRIQUES){id_parent}>
		<h2>#TITRE</h2>
		<BOUCLE_listDocs(DOCUMENTS){id_rubrique}{extension==jpg|png|gif|svg}{mode=document}>
		 #LOGO_DOCUMENT
		</BOUCLE_listDocs>
		<hr class="clear">
		</BOUCLE_filtreSecteur>

		<!-- Here emmet don't work div>ul>li : return :
		div>ul<div></div>
		-->
		
	</div>
<!-- Here emmet work div>ul>li : return :
<div>
	<ul>
		<li></li>
	</ul>
</div>
	-->

	<aside>
		<!-- Here emmet don't work div>ul>li partialy : return :
		div>ul<div></div>
	-->
		
	</aside>

	[<footer>
	<!-- Here emmet don't work div>ul>li : partialy : return
		div<div>
		<ul></ul>
	</div>
	-->

		<div class="notes"><h2><:info_notes:></h2>(#NOTES)</div>
	</footer>]
<!-- Here emmet work div>ul>li : return
<div>
	<ul>
		<li></li>
	</ul>
</div>
-->

</section>
</BOUCLE_content>
```

**Comment by Defman21** [comment]
What kind of file format are you using in Komodo to edit this file? I assume XML?

**Comment by mistergraphx** [comment]
No html or html5, the extension of the file is .html

**Comment by Defman21** [comment]
Reproduced. There are some problems with tabs somewhere in my logic code.

**Comment by Defman21** [comment]
This is a bug in Komodo: https://github.com/Komodo/KomodoEdit/issues/2351

**Comment by mistergraphx** [comment]
Hi, thanks for your answer and sorry for the time you've spend to investigate.

**Comment by Defman21** [comment]
No worries :)

**Comment by Defman21** [comment]
This should be fixed in Komodo 10.2.1.

**Comment by mistergraphx** [comment]
Yes it's ok ;-) thanks for your interest and addon
